### PR TITLE
CI tests: set up a few CI tests for kubernetes-csi/csi-driver-smb

### PR DESCRIPTION
**Pull request by Sakuralbj**
[Addressing](https://github.com/kubernetes-csi/csi-driver-smb/issues/54)
@andyzhangx

### CODE REVIEW

File: config/jobs/kubernetes-csi/csi-driver-smb/csi-driver-smb-config.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
+presubmits:
+  kubernetes-csi/csi-driver-smb:
+  - name: pull-csi-driver-smb-verify
```
Comment (msau42) [inline config/jobs/kubernetes-csi/csi-driver-smb/csi-driver-smb-config.yaml:3]: We actually have a script that generates pull and CI jobs for all the kubernetes-csi repos.

It involves ".prow.sh" with "UNIT", which already does build, verify, and unit tests. Do you want to use that instead?

https://github.com/kubernetes/test-infra/blob/c212d67a058661eb01c9bce29c1672826a2cddd8/config/jobs/kubernetes-csi/gen-jobs.sh#L390

Unfortunately the CI part is a bit hardcoded to the hostpath driver, so we would need to do some additional work to make it work for other drivers too cc @pohly

Comment (andyzhangx) [reply config/jobs/kubernetes-csi/csi-driver-smb/csi-driver-smb-config.yaml:3]: @msau42 shall we merge this PR first? we may modify this yaml in next iteration?

Comment (msau42) [reply config/jobs/kubernetes-csi/csi-driver-smb/csi-driver-smb-config.yaml:3]: Sure that's fine

Comment (andyzhangx) [reply config/jobs/kubernetes-csi/csi-driver-smb/csi-driver-smb-config.yaml:3]: thanks, filed a new issue to track this: https://github.com/kubernetes-csi/csi-driver-smb/issues/56

**Pull request merged**